projects/wasabix/index.js

```diff
@@ -507,12 +639,147 @@ async function bsc(timestamp, block, chainBlocks) {
     };
 }
 
+//polygon
+function getPolygonAddress(address) {
+    return `polygon:${address}`
+}
+
+async function polygon(timestamp, block, chainBlocks) {
+  let balances = {};
+  let tvl = 0;
+
+  //locker
+  const wasabiAddr = tokensPolygon.wasabi.address
+  const tokenStaked = await sdk.api.erc20.balanceOf({
+      owner: contractsPolygon.votingEscrow.address,
+      target: wasabiAddr,
+      chain: 'polygon',
+      block: chainBlocks['polygon']      
+  });
+  
+  const wasabiPrice = await getWasabiPrice('polygon', chainBlocks, 1)
```

Comment:
is this needed? Could just return wasabi directly and we'll get price from coingecko